src/Mod/PartDesign/TestPartDesignApp.py

```diff
@@ -143,27 +136,61 @@ def testMirroredCase(self):
         self.Rect.setDatum(10,App.Units.Quantity('1.0 mm'))
         self.Doc.recompute()
         self.Pad = self.Doc.addObject("PartDesign::Pad","Pad")
-        try:
-            self.Pad.Profile = self.Rect
-        except AttributeError:
-            self.Pad.Sketch = self.Rect
+        self.Pad.Profile = self.Rect
         self.Pad.Length = 1
-        try:
-            self.Body.addObject(self.Pad)
-        except AttributeError:
-            pass
+        self.Body.addObject(self.Pad)
         self.Doc.recompute()
         self.Mirrored = self.Doc.addObject("PartDesign::Mirrored","Mirrored")
         self.Mirrored.Originals = [self.Pad]
         self.Mirrored.MirrorPlane = (self.Rect, ["V_Axis"])
-        try:
-            self.Body.addObject(self.Mirrored)
-        except AttributeError:
-            pass
+        self.Body.addObject(self.Mirrored)
         self.Doc.recompute()
-        self.failUnless(self.Mirrored.Shape.Volume < 2.0)
+        self.failUnless(self.Mirrored.Shape.Volume == 1.9999999999999993)
```

Comment:
Exact comparison with floating numbers is a bad thing because it's simply too fragile. On my local system this unit test fails now. Also note that functions like failUnless are deprecated and should be replaced with their assert* equivalents.
In this case you should use assertAlmostEqual as described here https://docs.python.org/2/library/unittest.html